src/bundle/DependencyInjection/Configuration.php

```diff
@@ -93,74 +93,94 @@ public function getConfigTreeBuilder()
                     ->arrayNode('admin')
                         ->children()
                             ->scalarNode('endpoint')
-                            ->info('Admin api endpoint')
-                            ->example('end_point: https://admin.net')
+                                ->info('Admin api endpoint')
+                                ->example('end_point: https://admin.net')
```

Comment:
those examples seems to be inconsistent. Once you provide them with `key:` and in other places without. I would stick with the latter one as it is how we do it everywhere.